packages/@webex/plugin-meetings/test/unit/spec/media/index.ts

```diff
@@ -45,6 +49,7 @@ describe('createMediaConnection', () => {
   };
   afterEach(() => {
     sinon.restore();
+    clock.uninstall()
```

Comment:
What fails if this isn't present?

Reply:
The issue was in RtcMetrics. We are calling setInterval but in the test there was no FakeTimer being used. So the test passes but terminal doesnot exit with code 0.